### PR TITLE
📝 Community driven `Websurfx` instances page 

### LINKED ISSUE #252: 📝 Community driven `Websurfx` instances page
### Description

Provide an `instances` page in the `docs` which list the instances provided by the community and the instances provided by the project. 

### Screenshots

_No response_

### Do you want to work on this issue?

Yes

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by neon-mmd**
## What does this PR do?

This PR provides an instances page in the `docs` which provides all the instances maintained by the `Websurfx` community and by the `Websurfx` project. 

## Why is this change important?

This PR is essential as this provides the user the ability to use and try the project by using the provided instances. 

## Author's checklist

- [x] Provide an instances page in the `docs`.
- [x] Bump the app version to `v0.21.5` 

## Related issues

Closes #252

### CODE REVIEW

File: docs/instances.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Instances 
+
+> To contribute your server instance checkout the contributing guide [here](https://github.com/neon-mmd/websurfx/blob/HEAD/CONTRIBUTING.md).
```
Comment (alamin655) [inline docs/instances.md]:
```suggestion
> To contribute your server instance, check out the contributing guide [here](https://github.com/neon-mmd/websurfx/blob/HEAD/CONTRIBUTING.md).
```

File: docs/instances.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+This page provides a list of `Websurfx` instances provided by us and our community. 
+
+|URL|Network|Version|Location|Behind Cloudflare?|Official|TLS|IPv6|Comment|
+|-|-|-|-|-|-|-|
+|https://alamin655-websurfx.hf.space/|www|v0.21.4|🇺🇸US||✅|✅|||
```
Comment (alamin655) [inline docs/instances.md]:
```suggestion
| URL                                       | Network | Version | Location | Behind Cloudflare? | Official | TLS | IPv6 | Comment |
|-------------------------------------------|---------|---------|----------|--------------------|----------|-----|------|---------|
| https://alamin655-websurfx.hf.space/      | www     | v0.21.4 | 🇮🇳 IN  |                    | ✅       | ✅  |      |         |
```

**Review by alamin655 — CHANGES_REQUESTED** [review]
Consider replacing `Official` with `Maintained by` to clarify the instance's maintainer. This change will enhance transparency in the list.

File: docs/instances.md (hunk 1 above)
Comment (neon-mmd) [reply docs/instances.md]: Thanks :heart: for taking the time reviewing this PR. I will make the suggested changes right away :rocket: :slightly_smiling_face: .

**Comment by neon-mmd** [comment]
> Consider replacing `Official` with `Maintained by` to clarify the instance's maintainer. This change will enhance transparency in the list.

I think I agree with your point, but I think we can have it more of an optional requirement because I think some people might not feel very great sharing their profile. What do you say ? :slightly_smiling_face:

**Review by alamin655 — APPROVED** [review]
Approved! Let's merge this in. 👍

**Pull request merged**